qa/tasks/cephfs/fuse_mount.py

```diff
@@ -466,11 +466,12 @@ def admin_socket(self, args):
                 try:
                     p = self.client_remote.run(args=
                         ['sudo', self._prefix + 'ceph', '--admin-daemon', asok_path] + args,
-                        stdout=StringIO(), stderr=StringIO(),
+                        stdout=StringIO(), stderr=StringIO(), wait=False,
                         timeout=(15*60))
+                    p.wait()
                     break
                 except CommandFailedError:
-                    if "Connection refused" in stderr.getvalue():
+                    if "Connection refused" in p.stderr.getvalue():
```

Comment:
nit:
```suggestion
                    if "connection refused" in p.stderr.getvalue().lower():
```

to avoid a mismatch due to case?

Reply:
sure, changed